middleware/package.json

```diff
@@ -23,6 +23,7 @@
     "@semapps/signature": "0.4.0-alpha.32",
     "@semapps/sparql-endpoint": "0.4.0-alpha.32",
     "@semapps/triplestore": "0.4.0-alpha.32",
+    "@semapps/void": "file:/home/sebastien/SemApps/semapps/src/middleware/packages/void",
```

Comment:
this local file here? is it intended

Reply:
C'est temporaire, le temps que le package soit publié. Sinon yarn link échoue. La PR n'est pas prête.